client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-item.js

```diff
@@ -36,8 +36,8 @@ export class LabelItem extends Component {
 			toolTipMessage = translate( 'Labels older than 30 days cannot be refunded.' );
 			disabled       = true;
 		} 
-		else if ( isLetter && 'usps' === carrierId ) {
-			toolTipMessage = translate( 'USPS letters are not eligible for refund.' );
+		else if ( 'usps' === carrierId && ! trackingId ) {
+			toolTipMessage = translate( 'USPS untracked labels are not eligible for refund.' );
```

Comment:
(minor) Thoughts on phrasing this as `Untracked USPS labels are not eligible for refund.`?
```suggestion
			toolTipMessage = translate( 'Untracked USPS labels are not eligible for refund.' );
```

Reply:
Done!

Reply:
Thanks for the changes!